.isort.cfg

```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-known_third_party=
+known_third_party=setuptools
```

Comment:
```suggestion
known_third_party=dateutil,setuptools
```

Reply:
Fixed. Could you check again? Thank you :)